dora/shell/src/main/java/alluxio/cli/fsadmin/report/SummaryOutput.java

```diff
@@ -0,0 +1,454 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.cli.fsadmin.report;
+
+import alluxio.cli.fsadmin.FileSystemAdminShellUtils;
+import alluxio.grpc.MasterInfo;
+import alluxio.grpc.MasterVersion;
+import alluxio.util.CommonUtils;
+import alluxio.util.FormatUtils;
+import alluxio.wire.BlockMasterInfo;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+
+/**
+ * An output class, describing the summary.
+ */
+public class SummaryOutput {
+  private String mMasterAddress;
+  private int mWebPort;
+  private int mRpcPort;
+  private String mStarted;
+  private String mUptime;
+  private String mVersion;
+  private boolean mSafeMode;
+
+  private List<String> mZookeeperAddress;
+  private boolean mUseZookeeper;
+  private List<String> mRaftJournalAddress;
+  private boolean mUseRaftJournal;
+  private List<SerializableMasterVersion> mMasterVersions;
+
+  private int mLiveWorkers;
+  private int mLostWorkers;
+  private Map<String, String> mTotalCapacityOnTiers;
+  private Map<String, String> mUsedCapacityOnTiers;
+  private String mFreeCapacity;
```

Comment:
some of these no longer make sense in 3.x. But it's fine to keep it as-is in this PR, and remove irrelevant parts later.